jdaviz/configs/default/plugins/model_fitting/tests/test_plugin.py

```diff
@@ -0,0 +1,101 @@
+"""
+Tests the features of the Model Fitting Plugin (Selecting model parameters, adding models, etc.)
+This does NOT test the actual fitting self (see test_fitting.py for that)
+"""
+
+import uuid
+
+import pytest
+
+from jdaviz.configs.default.plugins.model_fitting.initializers import MODELS
+
+
+def test_default_model_labels(specviz_helper, spectrum1d):
+    specviz_helper.load_data(spectrum1d)
+    modelfit_plugin = specviz_helper.app.get_tray_item_from_name('g-model-fitting')
+    # By default, the spectral region should be the entire spectrum
+    assert modelfit_plugin.spectral_subset_selected == "Entire Spectrum"
+
+    for model in MODELS:
+        # Check that the auto label is set correctly (or at least the first character matches)
+        # BlackBody and Polynomial labels behave differently, so check only the first character
+        modelfit_plugin.comp_selected = model
+        assert modelfit_plugin.comp_label[0] == model[0]
+
+        # Test component label increments by default
+        previous_label = modelfit_plugin.comp_label
+        modelfit_plugin.vue_add_model()
+        try:
+            assert modelfit_plugin.comp_label == previous_label + "_1"
+        except AssertionError:
+            # Both Linear and Lortenz have the same default label
```

Comment:
is this something we want to change?  Maybe having Lorentz abbreviated to "Lo"?

Reply:
I was thinking that too. Not sure who would make that call?

Reply:
Looking at the code... it was already supposed to be `Lo`, but I accidentally had `"Lorentz"` instead of `"Lorentz1D"` (see `model_fitting.py _update_comp_label_default`), so I say make that fix and include it in this PR!  The test pulled through and found a bug!

Reply:
Before I make the change, do we want this to be the precedent? All new models should use the minimum number of unique characters for the default label? I can make that assumption in the test if we'd like?

Reply:
I don't think we need to actually test for that... but I guess if we keep expanding support for more models, we might eventually need a better "policy".  Right now the logic is the first letter, with three exceptions: Lorentz1D -> Lo (once you fix my typo, at least), BlackBody -> BB, PowerLaw -> PL.

Reply:
👍 I'll defer for now then!